src/components/elements/wysiwyg.tsx

```diff
@@ -58,7 +58,7 @@ const options: HTMLReactParserOptions = {
           return cleanMediaMarkup(domNode);
 
         case 'p':
-          nodeProps.className = twMerge(nodeProps.className, 'max-w-[100ch]');
+          nodeProps.className = twMerge(nodeProps.className, 'max-w-[100ch] leading-relaxed');
```

Comment:
Is this desired for every `<p>` tag? If you want to affect just the image gallery, that would be done somewhere else

Reply:
I did want it everywhere but I will double check. Thanks for pointing it out.

Reply:
@pookmish I did double check what we have on amptesting, and the wysiwyg are 2.1rem with 1.7 line-height.  I can follow up more with Design if you'd like.

Reply:
That's ok, just wanted to make sure it was intentional.

Reply:
In a way it should be changed in Decanter or in the tailwind config. But it's not really worth the effort right now.